csrankings-d.csv

```diff
@@ -480,7 +480,7 @@ David D. Jensen,University of Massachusetts Amherst,https://people.cs.umass.edu/
 David Dagan Feng,University of Sydney,http://sydney.edu.au/engineering/it/about/people/staff/feng.shtml,89py58oAAAAJ
 David Daniel Cox,Harvard University,http://www.coxlab.org,Pb5mcIYAAAAJ
 David Dapeng Zhang,The Hong Kong Polytechnic University,https://www4.comp.polyu.edu.hk/~csdzhang,IOagLnEAAAAJ
-David Darais,University of Vermont,http://david.darais.com,_0iLGqEAAAAJ
+David Darais,Galois,http://david.darais.com,_0iLGqEAAAAJ

```

Comment:
Please move this entry to `old/industry.csv`, with the affiliation at the end of the line. Thanks.